latextools_utils/cache.py

```diff
@@ -55,313 +68,577 @@ def hash_digest(text):
     return hash_result.hexdigest()
 
 
-def delete_local_cache(tex_root):
-    """
-    Removes the local cache folder and the local cache files
-    """
-    print(u"Deleting local cache for '{0}'.".format(repr(tex_root)))
-    local_cache_paths = [_hidden_local_cache_path(),
-                         _local_cache_path(tex_root)]
-    for cache_path in local_cache_paths:
-        if os.path.exists(cache_path):
-            print(u"Delete local cache folder '{0}'".format(repr(cache_path)))
-            shutil.rmtree(cache_path)
-
-
-def invalidate_local_cache(cache_path):
-    """
-    Invalidates the local cache by removing the cache folders
-    """
-    if os.path.exists(cache_path):
-        print(u"Invalidate local cache '{0}'.".format(repr(cache_path)))
-        shutil.rmtree(cache_path)
-
-
-def cache(tex_root, name, generate):
```

Comment:
I would like to keep `cache`, `read`, and `write` to easy access the local cache.